### DISCUSSION TITLE
The green Publish button is way too attractive

### DISCUSSION THREAD

**Discussion by chevdor** [community/community#6380] [Other Feature Feedback, Questions, & Ideas]
When preparing a new release or prerelease, the user is presented with the following UI (I am using the dark theme obviously):

![image](https://user-images.githubusercontent.com/738724/137519173-d14dc7a6-3d6e-440d-91d4-a1e26fee778b.png)

NOTE: the pre-release checkbox here is not relevant to the topic.

So as a user, I prepare a new release and when comes the time to save my draft, I go 🧲 for the big 🧲 green 🧲 button 🧲 to save as I don't want to cancel and lose my changes...

And just like that, unlike my wish to **only** save, I just published my release, created a tag, started Github actions, built docker images and published them, notified the people subscribed to new releasese etc... whereas I did not intend to do that yet.

This green button should protected **at least** by some sort of confirmation to prevent unwanted publication of drafts. I would not mind having to confirm with 2FA at that stage before releasing for instance.

It may also make sense on the UI side, to not use grey for a the `Save` button.

**Comment by MylesBorins** [comment]
Hey @chevdor. This was not something that was changed in the recent redesign. In most cases that people are creating releases they are publishing when they write the notes, no saving a draft (this comes from user research). If we do revisit the design of the publish page in the future we'll keep this in mind

**Reply by chevdor** [reply]
Hi @MylesBorins,

Indeed I am not saing this has changed recently, I just ran into the issue enough to take the time and bring it back here. 

While I can imagine that supporting 2FA can be quite some work, I think that adding a confirmation popup and making the color scheme clearer to the user will not bother much those fitting the most cases you mention and save the other from accidental and hardly reversible publishing. Those are not changes requiring days of effort to implement and test.

If there is a better place to report this issue, please let me know.

**Comment by chevdor** [comment]
There is another issue with this button.

The button seems to be primary so it will trigger when the user presses enter.

If you for instance Edit the title of your release or its tag, branch, etc... (the publish button may not even be in sight) and ever bounce on the Enter key (ie press it twice), then without any confirmation, this release you were working on is gone and the whole CI associated with it triggers....

This button would **really** deserve a confirmation popup.
